### DISCUSSION TITLE
Cannot deserialize value of type `int` from String "Access denied for user 'root@192.168.129.100' (using password: YES)": not a valid `int` value at [Source: (String)""Access denied for user 'root@192.168.129.100' (using password: YES)""

### DISCUSSION THREAD

**Discussion by 442496146** [apache/doris#17523] [A - General / Q&A]
![image](https://user-images.githubusercontent.com/40668945/223385329-d8f989c9-c0cf-4a97-b7b0-ce1d66044113.png)
 sql
CREATE TABLE cdc_mysql_source (
 `id` STRING,
 `order_code` STRING,
 `line_code` STRING,
 `remark` STRING,
 `unit_no` STRING,
 `unit_name` STRING,
 `price` STRING,
 PRIMARY KEY(`id`) NOT ENFORCED
) WITH (
 'connector' = 'mysql-cdc',
 'hostname' = '192.168.129.222',
 'port' = '3310',
 'username' = 'root',
 'password' = 'xxxx2022',
 'database-name' = 'test',
 'table-name' = 't_test'
);

SET 'execution.checkpointing.interval' = '10s';
CREATE TABLE doris_sink (
 `id` STRING,
 `order_code` STRING,
 `line_code` STRING ,
 `remark` STRING ,
 `unit_no` STRING,
 `unit_name` STRING,
 `price` STRING
) 
WITH (
  'connector' = 'doris',
  'fenodes' = '192.168.129.101:8030',
  'table.identifier' = 'doristest.ods_t_test',
  'username' = 'root',
  'password' = '0000000',
  'sink.properties.format' = 'json',
  'sink.properties.read_json_by_line' = 'true',
  'sink.enable-delete' = 'true',
  'sink.label-prefix' = 'doris_label'
);
sql
insert into doris_sink select id,order_code,line_code,remark,unit_no,unit_name,price from cdc_mysql_source;

flink-doris-connector-1.16

**Comment by AhmadrezaHamidi** [comment]
Based on the SQL statements provided, it appears to be a Flink job configuration that sets up two tables and inserts data from the first table into the second table.

The first table, "cdc_mysql_source", is created using the MySQL-CDC connector, which connects to a MySQL database at IP address 192.168.129.222 on port 3310. The table has columns for id, order_code, line_code, remark, unit_no, unit_name, and price, with id set as the primary key. Checkpointing is enabled with an interval of 10 seconds.

The second table, "doris_sink", is created using the Doris connector, which connects to a Doris cluster at IP address 192.168.129.101 on port 8030. The table has the same column names and data types as the first table. The "sink" properties are set to format the data as JSON, read the JSON data line by line, enable deletion of rows, and prefix labels with "doris_label".

The last statement is an SQL INSERT statement that selects data from the "cdc_mysql_source" table and inserts it into the "doris_sink" table.

The version of the Flink-Doris connector used is not specified in the given SQL statements.
